test/scholar/linear/svm_test.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Scholar.Linear.SVMTest do
+  use Scholar.Case, async: true
+  alias Scholar.Linear.SVM
+  doctest SVM
+
+  test "Iris Data Set - multinomial classification svm test" do
+    Nx.Defn.default_options(compiler: EXLA)
+
+    {x_train, x_test, y_train, y_test} = Datasets.get(:iris) |> Nx.backend_transfer(EXLA.Backend)
+    y_train = Nx.dot(y_train, Nx.iota({3}, backend: EXLA.Backend))
+    y_test = Nx.dot(y_test, Nx.iota({3}, backend: EXLA.Backend))
```

Comment:
This module doesn't exist anymore and should be replaced with the function that was previously in this PR